packs/data/extinction-curse-bestiary.db/wight-cultist.json

```
@@ -1405,7 +1563,7 @@
             "alignment": {
                 "value": "NE"
             },
-            "blurb": "",
+            "blurb": "Wight variant",
```

```suggestion
            "blurb": "Variant wight",
```
This is almost certainly the wording in the actual book.